src/web/pages/cves/details.jsx

```diff
@@ -25,6 +25,7 @@ import TableData from 'web/components/table/data';
 import TableRow from 'web/components/table/row';
 
 import PropTypes from 'web/utils/proptypes';
+import useGmp from "web/utils/useGmp.jsx";
```

Comment:
Imports should not use an extension. @daniele-mng could we have a eslint rule for that too?

```suggestion
import useGmp from "web/utils/useGmp";
```